### PR TITLE
New flag: -l SUBSET

### PR DESCRIPTION
**Pull request by sitamet**
New flag: -l SUBSET, --limit=SUBSET further limit selected hosts to an additional pattern.

Why:
Node v4 does not accept a spawn with fake-command-params arrays e.g ['-i','hostfile -l myhost']
Now in order to pass additional params we have to set them properly e.g ['-i','hostfile', '-l', 'myhost']

### CODE REVIEW

**Comment by macobo** [comment]
:+1: This seems solid. @shaharke

File: docs/index.html [hunk 1]
```diff
@@ -179,7 +179,12 @@ <h3 id="supported-flags">Supported Flags</h3>
         
           <div class='highlight'><pre>command.inventory(<span class="hljs-string">'/etc/ansible/hosts'</span>)</pre></div>
         
-      
+
```
Comment (shaharke) [inline docs/index.html]: No need to change the actual HTML file only examples/index.js. Docs are auto generated.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-ansible",
-  "version": "0.5.2",
+  "version": "0.5.3",
```
Comment (shaharke) [inline package.json]: Don't bump version. Leave that to me.

**Comment by shaharke** [comment]
Looks solid except for the comments I've added. Please fix and I'll merge and publish.

**Comment by macobo** [comment]
Comments on commits don't get sent to participants. There's a merge conflict in the code @sitamet.

**Comment by shaharke** [comment]
@macobo thanks. I really didn't see the comment. @sitamet can you rebase and fix the conflicts?

**Comment by macobo** [comment]
:+1: thanks!

**Comment by macobo** [comment]
:) This has been rebased. @sitamet @shaharke probably doesn't see your rebases unless you leave a comment.

**Comment by shaharke** [comment]
@macobo thanks. It might take me a while to merge. sorry for the delays.

**Comment by macobo** [comment]
Any way I can speed this up? This missing feature would make my life much much easier for usage in chatops. :)

**Comment by macobo** [comment]
Thank you!

**Pull request merged**